api/tests/Feature/PoolCandidatesPaginatedTest.php

```diff
@@ -167,7 +167,6 @@ public function testUnassociatedProcessOperatorCannotViewAnyApplications(): void
         $this->assertPaginatedResponse($processOperator, 0, []);
     }
 
-    // TO DO: Update in #10364
     public function testCommunityRecruiterCannotViewAnyApplications(): void
```

Comment:
These two tests should be updated to confirm that community recruiters and admins _can_ see any pools created in their communities, but not in other communities.

Reply:
It's `PoolCandidatesPaginatedTest.php` so I am assuming you mean applications and not pools. 

Turns out some tweaking was needed for it to work, updated and added tests in 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11070/commits/4d33d4f7ccbb6eb1bc2467cbbda8920944d0cd36
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11070/commits/ba3b7343589763a2bffae19066b4debfc9323351